flow360/component/simulation/translator/solver_translator.py

```diff
@@ -526,6 +534,36 @@ def process_output_fields_for_udf(input_params):
 
     return generated_udfs
 
+def translate_streamtrace_output(output_params: list):
```

Comment:
add test in test in: https://github.com/flexcompute/Flow360/blob/develop/tests/simulation/translator/test_output_translation.py